rules/windows/collection_posh_screen_grabber.toml

```diff
@@ -14,6 +14,61 @@ index = ["winlogbeat-*", "logs-windows.*"]
 language = "kuery"
 license = "Elastic License v2"
 name = "PowerShell Suspicious Script with Screenshot Capabilities"
+note = """## Triage and analysis
+
+### Investigating PowerShell Suspicious Script with Screenshot Capabilities
+
+PowerShell is one of the main tools system administrators use for automation, report routines, and other tasks, making
+it available for use in various environments, creating an attractive way for attackers to execute code.
+
+Attackers can abuse PowerShell capabilities to take screen captures of the desktop to gather information over the course
+of an operation.
+
+#### Possible investigation steps:
+
+- Examine script content that triggered the detection. 
+- Investigate script execution chain (parent process tree).
+- Inspect any file or network events from the suspicious PowerShell host process instance.
+- Investigate other alerts related to the user/host in the last 48 hours.
```

Comment:
Making this recommendation so that it's consistent with my edits in your other PR. Just leaving one this time, so feel free to ignore or apply to all other instances if you accept the change.